src/devcenter/azext_devcenter/generated/_params.py

```diff
@@ -0,0 +1,730 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+# Generated control plane params from autorest
+# DO NOT UPDATE THIS FILE, used to track changes from autorest generation
+# Update the _params.py under the manual
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import (
+    get_default_location_from_resource_group,
+    validate_file_or_dict
+)
+from azext_devcenter.action import (
+    AddGitHub,
+    AddImageReference,
+    AddSku
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('devcenter dev-center list') as c:
```

Comment:
Why the command name contains double devcenter?

Reply:
Our RP name is Microsoft.DevCenter and we have a resource named "DevCenter" as well. We have gotten feedback about this "double" naming and this still under review.